src/app.tsx

```diff
@@ -122,14 +127,16 @@ async function main() {
 
   let { Player, Menu, Platform } = Spicetify;
   let mainElem = document.querySelector('.main-view-container__scroll-node-child');
+  let shortcutsElem = document.querySelector('.view-homeShortcutsGrid-grid');
 
-  while (!Player || !Menu || !Platform || !mainElem) {
+  while (!Player || !Menu || !Platform || !mainElem || !shortcutsElem) {
```

Comment:
I also don't think we can put this here, since the app may not start on the homepage (and their Spotify may not even have it). 

It might be a better idea to add a custom element to check for on the homepage down in the `listenThenApply` function, like how it does for the `/search` page.